### PR TITLE
Master

### PR DESCRIPTION
**Pull request by FM1337**
Added compliments plugin to dan!

### CODE REVIEW

File: addons/commands/fun/compliment.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+command(['compliment'])
+    ->helpText('Compliments someone.')
+    ->handler(function(\Dan\Irc\Connection $connection, Channel $channel, User $user, $message) {
+        $websites = array('http://toykeeper.net/programs/mad/compliments', 'http://www.chainofgood.co.uk/passiton', 'http://www.madsci.org/cgi-bin/cgiwrap/~lynn/jardin/SCG');
+        $website = array_random($websites);
+        $classes = array('large', 'medium', 'small');
+        $class = array_random($classes);
+        $compliment = Web::xpath($website);
+        if ($website == "http://www.chainofgood.co.uk/passiton"){
```
Comment (UclCommander) [inline addons/commands/fun/compliment.php]: Line break above this statement

File: addons/commands/fun/compliment.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+use Dan\Irc\Location\Channel;
+use Dan\Irc\Location\User;
+use Dan\Support\Web;
+
+command(['compliment'])
+    ->helpText('Compliments someone.')
+    ->handler(function(\Dan\Irc\Connection $connection, Channel $channel, User $user, $message) {
+        $websites = array('http://toykeeper.net/programs/mad/compliments', 'http://www.chainofgood.co.uk/passiton', 'http://www.madsci.org/cgi-bin/cgiwrap/~lynn/jardin/SCG');
+        $website = array_random($websites);
+        $classes = array('large', 'medium', 'small');
```
Comment (UclCommander) [inline addons/commands/fun/compliment.php]: Use short array syntax

File: addons/commands/fun/compliment.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        else if ($website == "http://www.madsci.org/cgi-bin/cgiwrap/~lynn/jardin/SCG"){
+        $compliment = cleanString($compliment->query('//h2')->item(0)->textContent);
+        }
+        else if ($website == "http://toykeeper.net/programs/mad/compliments"){
+        $compliment = cleanString($compliment->query('//*[@class="blurb_title_1"]')->item(0)->textContent);
+        }
+/* In case the chosen website fails top load */
+        else{
+        $compliment = "You are very lovely and everyone cares about you!";
```
Comment (UclCommander) [inline addons/commands/fun/compliment.php]: Intent these and put else / else if on the same line as `}`

**Comment by UclCommander** [comment]
Please correct the spacing and styling according to StyleCI.

**Comment by FM1337** [comment]
Compliments is now styleci ready.

**Pull request merged**